operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -104,13 +109,20 @@
   }
 
   private static boolean isPatchableItem(Map.Entry<String, String> entry) {
-    return isCustomerItem(entry) || entry.getKey().equals(INTROSPECTION_STATE_LABEL);
+    return isCustomerItem(entry) || PATCHABLE_OPERATOR_KEYS.contains(entry.getKey());
   }
 
+  private static final Set<String> PATCHABLE_OPERATOR_KEYS
+        = Set.of(INTROSPECTION_STATE_LABEL, OPERATOR_VERSION, MODEL_IN_IMAGE_DOMAINZIP_HASH, SHA256_ANNOTATION);
```

Comment:
@jshum2479, can you please confirm whether we would want to patch MODEL_IN_IMAGE_DOMAINZIP_HASH? The use case would be that we've decided not to roll an existing pod, but the pod's value for this label doesn't match the pod model. I believe this means that we are in a dynamic update use case and the dynamic update was successful. Given that, it's probably fine to either patch this value or not patch this value; however, I'd like your opinion. I'm confident that Russ' change to patch OPERATOR_VERSION and SHA256_ANNOTATION are appropriate.

Reply:
We need patch it for the use case you described,  the code that checks for whether domain zip has changed to reuse the pod / patch the pod depends on the domain zip hash label and the result coming from the introspector